suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -686,3 +686,13 @@ export const selectSelectedDeviceName = (state: DeviceRootState) => {
     const selectedDevice = selectDevice(state);
     return selectDeviceName(state, selectedDevice?.id);
 };
+
+export const selectSelectedDeviceFirmwareVersion = (state: DeviceRootState) => {
+    const selectedDevice = selectDevice(state);
+    return selectedDevice?.firmwareRelease?.release.version;
```

Comment:
Not much of a difference other than semantics, but if there is no value, I'd prefer returning `null` as in the other selectors. If you change this, apply also to `selectSelectedDeviceModel` please 

```suggestion
    return selectedDevice?.firmwareRelease?.release.version ?? null;
```

Reply:
f0cbf4f17c